dev-app/src/screens/LocationListScreen.tsx

```diff
@@ -22,6 +22,7 @@ export default function LocationListScreen() {
 
   const { getLocations, loading } = useStripeTerminal();
   const [list, setList] = useState<Location[]>([]);
+  const dummyLocations: Location[] = [{ id: "ABCD", displayName: "Bad Location", livemode: false }, { id: "tml_AbC2def4GhIjkL", displayName: "Wrong User Location", livemode: false }]
```

Comment:
this line is still causing lint failures in the build, `npx prettier dev-app/src/screens/LocationListScreen.tsx --write` should fix it